src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,23 @@ const createCustomerValidator = () => {
     });
 };
 
+const getCustomersAllValidator = () => {
+    return celebrate({
+        [Segments.QUERY]: Joi.object().keys({
+            offset: Joi.string(),
```

Comment:
Poderia aqui já fazer uma validação do campo ser número inteiro, e também colocar o valor padrão caso o usuário não informe nenhum valor.
```suggestion
            offset: Joi
                .number()
                .integer()
                .min(0)
                default(0),
```